shared/src/test/diff/mlscript/Arrays.mls

```diff
@@ -137,7 +143,7 @@ ty2A = ty2B
 //│ ╔══[ERROR] Type mismatch in def definition:
 //│ ║  l.+1: 	ty2A = ty2B
```

Comment:
line number is weird

Reply:
This is just the way line numbers local to the current block are printed with option `:ShowRelativeLineNums`. It's used to minimize diffs due to line number changes in errors. We should really use that approach more often!